canonical-websites/www.ubuntu.com
# Complete update of /support/plans-and-pricing page

## Done

Complete content re-write of the /support/plans-and-pricing page. Separated the price tables out into re-usable includes.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/support/plans-and-pricing](http://0.0.0.0:8001/support/plans-and-pricing)
- Check visually and against the [copydoc](https://docs.google.com/document/d/1R7FRZbfx_5YHFeZUD93HgT8RW4OCbbK4blOulB4D6EE/edit#heading=h.e7s406oqmgi9)


## Issue / Card

Fixes #2198
